src/main/java/core/basesyntax/classes/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.classes;
```

Comment:
all of our classes are classes :)
```suggestion
package core.basesyntax.model;
```